imgui-app/src/main/java/imgui/app/Window.java

```diff
@@ -163,6 +164,7 @@ protected void startFrame() {
         GL32.glClear(GL32.GL_COLOR_BUFFER_BIT | GL32.GL_DEPTH_BUFFER_BIT);
         imGuiGlfw.newFrame();
         ImGui.newFrame();
+        ImGuizmo.beginFrame();
```

Comment:
`Window` is a general class to implement ImGui API. So other stuff like extension usage should be implemented in user specific code. Method `startFrame` could be overriden with additional logic. This line should be removed from here.